app/alerts.py

```diff
@@ -88,50 +91,105 @@ def build_alerts_geojson():
 
 
 # ------------------------------------------------------------------------------
-# Cameras
-# The following block is dedicated to fetching information about cameras and displaying them on the map
-
+# Sites markers
 # Fetching the positions of detection units in a given department
-def get_camera_positions(dpt_code=None):
+def build_sites_markers(dpt_code=None):
 
     # As long as the user does not click on a department, dpt_code is None and we return no device
-    if not dpt_code:
-        return None
+    # if not dpt_code:
+    #     return None
 
     # We read the csv file that locates the cameras and filter for the department of interest
     camera_positions = pd.read_csv(Path(__file__).parent.joinpath('data', 'cameras.csv'), ';')
-    camera_positions = camera_positions[camera_positions['Département'] == int(dpt_code)].copy()
+    # camera_positions = camera_positions[camera_positions['Département'] == int(dpt_code)].copy()
 
-    # We build a list of dictionaries containing the info of each camera
+    # We build a list of markers containing the info of each site/camera
     markers = []
-    for _, row in camera_positions.iterrows():
+    for i, row in camera_positions.iterrows():
         lat = row['Latitude']
         lon = row['Longitude']
-        area = row['Tours']
-        alert_codis = row['Connexion Alerte CODIS']
+        site_name = row['Tours']
         nb_device = row['Nombres Devices']
-        popup = ["Ville: {} <br>\
-                 Connexion Alerte CODIS: {} <br>\
-                 Nombres Devices: {}".format(area, alert_codis, nb_device)]
-        markers.append(dict(lat=lat,
-                            lon=lon,
-                            area=area,
-                            alert_codis=alert_codis,
-                            nb_device=nb_device,
-                            popup=popup))
+        markers.append(dl.Marker(id=f'site_{i}',    # Necessary to set an id for each marker to reteive callbacks
```

Comment:
```suggestion
        markers.append(dl.Marker(id=f'site_{i}',    # Necessary to set an id for each marker to receive callbacks
```

Small typo in the comment. 

Just to confirm and if I am not mistaken, the `site` table that we will call via the API contains a `site_id` which we will be able to use instead of the index of the for loop here, right?

Reply:
yes I think you're right @pechouc, the f-string id was the previous way of getting a marker, from now on we should have a real site_id